osmosis-1/osmosis.zone_chains.json

```diff
@@ -6,23 +6,14 @@
       "rpc": "https://rpc-osmosis.keplr.app",
       "rest": "https://lcd-osmosis.keplr.app",
       "explorer_tx_url": "https://www.mintscan.io/osmosis/txs/${txHash}",
-      "keplr_features": [
-        "ibc-go",
-        "ibc-transfer",
-        "cosmwasm",
-        "wasmd_0.24+",
-        "osmosis-txfees"
-      ]
+      "keplr_features": ["ibc-go", "ibc-transfer", "cosmwasm", "wasmd_0.24+", "osmosis-txfees"]
```

Comment:
Can you please undo the update of the formatting so it doesn't change the whole file?
[
  "new line",
  "for each"
]